cilium/ebpf.io-summit
# Style updates

**Describe what changes this pull request brings**
This pull request brings the changes below:
- [x] remove the Agenda section
- [x] adjust styles for Information section
- [x] add button Create your agenda in Schedule




**Steps to test**
1. Open [the page](https://deploy-preview-7--ebpf-summit-2021.netlify.app/summit-2021)
2. Confirm that everything looks and works as expected


**Screenshots**
![image](https://user-images.githubusercontent.com/48465000/128332113-d04cab3e-346f-4356-8bf7-f0f7749e01b1.png)

![image](https://user-images.githubusercontent.com/48465000/128332185-7bf25722-dfe4-441f-9c76-536c070c6c41.png)

![image](https://user-images.githubusercontent.com/48465000/128332234-b78ae5b1-b273-4bc8-abf1-0ddde3d4ac38.png)



**Checklist**
